src/main/groovy/de/dkfz/roddy/tools/DateTimeHelper.groovy

```diff
@@ -0,0 +1,56 @@
+package de.dkfz.roddy.tools
+
+import groovy.transform.CompileStatic
+
+import java.time.Duration
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+@CompileStatic
+class DateTimeHelper {
+
+    private final DateTimeFormatter DATE_PATTERN
+
+    DateTimeHelper(String dateParserPatternPattern = null, ZoneId timeZoneId = ZoneId.systemDefault()) {
+        if (!dateParserPatternPattern)
+            DATE_PATTERN = DateTimeFormatter.ISO_DATE_TIME
+        else
+            this.DATE_PATTERN = DateTimeFormatter
+                    .ofPattern(dateParserPatternPattern)
+                    .withLocale(Locale.ENGLISH)
```

Comment:
Why fix the locale as ENGLISH?

Reply:
Actually I don't know, I reused existing code. @askask , @fkaercher do you know, why you put this in?